core/tests/ts-integration/tests/system.test.ts

```diff
@@ -311,6 +311,12 @@ describe('System behavior checks', () => {
         ).toBeAccepted();
     });
 
+    test('Gas per pubdata byte getter should work', async () => {
+        const systemContextArtifact = getTestContract('ISystemContext');
+        const systemContext = new ethers.Contract(SYSTEM_CONTEXT_ADDRESS, systemContextArtifact.abi, alice.provider);
+        console.log(await systemContext.gasPerPubdataByte());
```

Comment:
Can the result be somehow asserted instead of being printed?

Reply:
definitely, sorry